internal/daemon/controller/handlers/accounts/grants_test.go

```diff
@@ -109,3 +121,2895 @@ func TestListPassword_Grants(t *testing.T) {
 		})
 	}
 }
+
+func TestGrants_GetAccounts(t *testing.T) {
+	t.Parallel()
+	ctx := context.TODO()
```

Comment:
I've been defaulting to `context.Background()`, but I've seen `context.TODO()` used in some places.

Which one should we align on, `TODO()` or `Background()`?